.github/workflows/build_wheel.yml

```
@@ -25,8 +25,9 @@ jobs:
 
       - name: Build wheels
         env:
-          CIBW_BUILD: "cp36-* cp37-* cp38-*"
-          CIBW_BEFORE_BUILD: pip install tensorflow && sed -i 's/libresolv.so.2"/libresolv.so.2", "libtensorflow_framework.so.2"/g' /opt/_internal/tools/lib/python*/site-packages/auditwheel/policy/policy.json
+          CIBW_BUILD: "cp36-* cp37-* cp38-* cp39-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: ghcr.io/njzjz/manylinux2010_x86_64_tensorflow
```

I don't have the access to `ghcr.io/deepmodeling/manylinux2010_x86_64_tensorflow` using @amcadmus's token... I need the access to such package to make it public. Also, using my own token, I cannot create a new container package.  It should be set on https://github.com/organizations/deepmodeling/settings/packages.

```suggestion
          CIBW_MANYLINUX_X86_64_IMAGE: ghcr.io/deepmodeling/manylinux2010_x86_64_tensorflow
```